src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -628,10 +628,6 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\test\\CadlRanchProjects\\azure\\client-generator-core\\access\\src\\Generated -n"
     },
-    "typespec-azure/client-generator-core/internal": {
-      "commandName": "Project",
-      "commandLineArgs": "--standalone $(SolutionDir)\\test\\CadlRanchProjects\\azure\\client-generator-core\\internal\\src\\Generated -n"
```

Comment:
Could you help to explain why we remove this test project?

Reply:
it is removed from cadl-ranch